packages/workbox-core/.npmignore

```diff
@@ -0,0 +1 @@
+browser.mjs
```

Comment:
Just checking: we can't use [`package.files`](https://docs.npmjs.com/files/package.json#files) to whitelist what we publish to `npm`, because that's going to include a bunch of other `.mjs` files, including some at the same level as `browser.mjs`. So it's easier for you to just blacklist that one file?